src/main/resources/locales/cy/translation/single-or-multiple-claim.json

```diff
@@ -0,0 +1,29 @@
+{
+  "title": "Lorem ipsum dolor sit",
+  "h1": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer convallis mi sed erat dapibus interdum non?",
+  "p1": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse metus lacus, convallis nec maximus in, vehicula at nunc. Sed facilisis accumsan dolor ac mattis.",
+  "p2": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc id tempus risus. Ut tortor ex, lacinia ac dui ac, porta dapibus sem. Duis pellentesque lacus non lectus tincidunt.",
+  "radio1": "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+  "radio2": "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+  "serviceName": "Lorem ipsum",
+  "guide": "Lorem",
+  "termsAndConditions": "Lorem ipsum dolor",
+  "cookiePolicy": "Lorem",
+  "contact": "Lorem ipsum",
+  "stage": "Lorem",
+  "errorSummaryTitle": "Lorem ipsum dolor sit",
+  "continue": "Lorem",
+  "back": "Lorem",
+  "contactUsTitle": "Lorem ipsum dolor sit",
+  "contactUsForHelp": "Lorem ipsum dolor sit amet<br>Lorem ipsum dolor<br>Lorem ipsum dolor sit amet<br>Lorem ipsum dolor<br>Lorem ipsum dolor sit amet, consectetur<br> <br><a href=\"#\" class=\"govuk-link\">Lorem ipsum dolor sit amet</a>",
+  "errors": {
+    "isASingleClaim": {
+      "required": "Lorem ipsum dolor"
+    }
+  },
+  "phaseBanner": {
+    "feedback": "Lorem ipsum dolor",
+    "additionalText": "sit amet, consectetur adipiscing elit.",
+    "languageToggle": "<a href=\"{currentUrl}{languageFlag}\" class=\"govuk-link language\">English</a>"
+  }
+}
```

Comment:
Majority of these things should be in common and template translation files for CY

Reply:
Moved the common translations to common and translation files.